posnums.c

```diff
@@ -0,0 +1,18 @@
+i#include <stdio.h>
```

Comment:
Furthermore, your code returned a segmentation fault. You should have at least attempted to get rid of the seg fault and add comments if you couldn't complete the rest of the assignment.